.vscode/settings.json

```diff
@@ -76,5 +76,6 @@
 	"[typescript]": {
 		"editor.defaultFormatter": "vscode.typescript-language-features"
 	},
-	"typescript.tsc.autoDetect": "off"
+	"typescript.tsc.autoDetect": "off",
+	"debug.javascript.warnOnLongPrediction": false
```

Comment:
Where is this coming from?

Reply:
came automatically on running, not sure how, just removing it